main.js

```diff
@@ -1,38 +1,82 @@
-                </div>
-                <div class="description">
-                    <div class="name-and-price flex-with-direction white-opacity">
-                        <h3 class="dish-name">${dish.title}</h3>
-                        <h3 class="dish-price">${dish.price}</h3>
+const getData = fetch(API_URL)
+    .then(resp => resp.json())
+    .then(data => {
+        /**
+         * renderDataToMenu() function
+         * renders data into main menu dynamically 
+         * as a parameter it takes data that's coming from api
+         */
+        const renderDataToMenu = (data) => {
+            data.forEach(dish => {
+                // creating a div element for each dish into main container
+                let cardDiv = document.createElement("div");
+                // appending each div to main container
+                mainMenu.appendChild(cardDiv);
+                // adding the class name for div
+                cardDiv.className = "card flex-with-direction";
+                // rendering the data into each div
+                cardDiv.innerHTML = `
+                    <div class="img">
+                        <img src="${dish.img}" alt="${dish.title}">
                     </div>
-                    <div class="dish-info">
-                        <p>${dish.desc}</p>
+                    <div class="description">
+                        <div class="name-and-price flex-with-direction white-opacity">
+                            <h3 class="dish-name">${dish.title}</h3>
+                            <h3 class="dish-price">${dish.price}</h3>
+                        </div>
+                        <div class="dish-info">
+                            <p>${dish.desc}</p>
+                        </div>
+                        <div class="add-to-cart">
+                            <button class="add-to-button">ADD TO CART</button>
+                        </div>
                     </div>
-                </div>
-            `
-        })
-    })
-}
-renderDataToMenu();
+                `
+            })
+        }
+
+        renderDataToMenu(data)
+
+        /**
+         * searchInfo function will get triggered once user clicks search button
+         * users input will be read and function will seach thru the data
+         * if word matches the title or the description of the dish then
+         * it will return the item to and searchedItems. By using this temp
+         * searchedItems as argument, renderDataToMenu(searchedItems) will be called 
+         * and the data found by search result will be displayed on the main menu.
+         */
+        const searchInfo = () => {
+
+            let searchedWord = searchInput.value;
+
+            let searchedItems = [];
+            data.forEach(dish => {
+                if (dish.title.includes(searchedWord) ||
+                    dish.desc.includes(searchedWord)) {
+                    searchedItems.push(dish)
+                }
+            })
```

Comment:
What do you think if we use .filter() here, instead of .forEach()? Try this:
`
let searchedItems = data.filter(dish => dish.title.includes(searchedWord) || dish.desc.includes(searchedWord))
`